horizon/config.py

```diff
@@ -103,6 +104,24 @@ def __new__(cls, prefix=None, is_model=True):
     # enable datadog APM tracing
     ENABLE_MONITORING = confi.bool("ENABLE_MONITORING", False)
 
+    ENABLE_OFFLINE_MODE = confi.bool(
+        "ENABLE_OFFLINE_MODE",
+        False,
+        description="if true, sidecar will use a file backup to restore configuration and policy data when cloud services are unavailable",
+    )
+
+    OFFLINE_MODE_BACKUP_PATH = confi.str(
+        "OFFLINE_MODE_BACKUP_PATH",
+        "/app/backup/sidecar_config.json",
```

Comment:
Where do you create this directory in case it doesn't exist ?
I don't think `open(filename,"w")` as f does that

Reply:
Yeah I don't.
In OPAL I do call `makedirs` I think.
I'll add it here as well